core/src/bin/do_auth_flow_and_refresh.rs

```diff
@@ -0,0 +1,25 @@
+use mongo_odbc_core::oidc_auth::*;
+
+#[tokio::main]
+async fn main() {
```

Comment:
This is only for testing, right?
What about we rename the files in bin something like "test_auth_flow_and_refresh.rs", "test_auth_flow.rs" and "test_rfc8252_http_server.rs". This makes it clear it is just for testing and nothing else.


Also, I think that these files are built and bundled in the driver when released. Should we use a feature flag to only build it when needed for testing?